WcaOnRails/config/environments/development.rb

```diff
@@ -2,6 +2,17 @@
 
 require "active_support/core_ext/integer/time"
 
+root_url = EnvVars.ROOT_URL
+unless root_url.present?
+  root_url = "http://localhost:3000"
+end
+root_url = URI.parse(root_url)
```

Comment:
```suggestion
root_url = URI.parse(EnvVars.ROOT_URL)
```

I believe we can simplify now as `ROOT_URL` has a default :)

https://github.com/thewca/worldcubeassociation.org/blob/fe6cf26b90f2d78dda44b318b765e2096767555c/WcaOnRails/env_vars.rb#L22-L37